apps/web/src/app/[locale]/(profile)/[username]/_components/dashboard/overview-tab.tsx

```diff
@@ -13,18 +14,25 @@ export async function OverviewTab({ user }: Props) {
   const hasBio = Boolean(user.bio) && user.bio.trim().length > 0;
 
   return (
-    <Card className="col-span-4 md:min-h-[calc(100vh_-_56px_-_6rem)]">
-      <CardHeader>
-        <CardTitle>Overview</CardTitle>
-      </CardHeader>
-      <CardContent>
-        <div className="w-full text-sm">
-          {Boolean(hasBio) && <Markdown>{user.bio}</Markdown>}
-          {Boolean(!hasBio) && (
-            <p className="text-muted-foreground">{user.name} does not have a bio.</p>
-          )}
-        </div>
-      </CardContent>
-    </Card>
+    <div className="col-span-4 flex flex-col gap-6 md:min-h-[calc(100vh_-_56px_-_6rem)]">
+      <div className="flex flex-col md:flex-row md:gap-6">
+        <ChallengesProgress />
+        {/* //Todo: Filling with void for now, may put contributions / something else */}
+        <div className="max-w-sm flex-grow" />
+      </div>
+      <Card className="flex-grow">
+        <CardHeader>
+          <CardTitle>Bio</CardTitle>
+        </CardHeader>
+        <CardContent>
+          <div className="w-full text-sm">
+            {Boolean(hasBio) && <Markdown>{user.bio}</Markdown>}
+            {Boolean(!hasBio) && (
```

Comment:
you shouldn't need the extra `Booleam` wrappers here.

Reply:
Not my part of code I think, anyways I'll fix. Thanks.